Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -635,7 +635,11 @@ Refer to the chapter "[Tampering and Reverse Engineering on iOS](0x06c-Reverse-E
 
 The goal of emulator detection is to increase the difficulty of running the app on an emulated device. This forces the reverse engineer to defeat the emulator checks or utilize the physical device, thereby barring the access required for large-scale device analysis.
 
-However, this is not a concern on iOS. As discussed in the section [Testing on the iOS Simulator](0x06b-Basic-Security-Testing.md "Testing on the iOS Simulator") in the basic security testing chapter, the only available simulator is the one that ships with Xcode. Simulator binaries are compiled to x86 code instead of ARM code and apps compiled for a real device (ARM architecture) don't run in the simulator. This makes the simulator useless for black box analysis and reverse engineering.
+Previously XCode was the only choice if a developer needed to mimic an iOS or iPadOS device. Mimic really is the best term to describe the level of emualtion that is really possible within XCode, hence emulator protection was not so much a concern regarding iOS apps in contrast to Android with a wide range of emulation choices available. With the release of [Corellium](https://www.corellium.com/), a publicly available iOS emulator, the playfield has shifted towards an emulation, still limited (i.e. IPAs need resigning), but as a typical SaaS offering, employing a vast number of devices is just a few mouse clicks away with the limiting factor just being available funds.
```

Comment:
Mimic really is the best term to describe the level of **emualtion** that is really possible within XCode
emulation <- spelling error.

Reply:
Thanks for reviewing, fixed typo.